odxtools/cli/compare.py

```diff
@@ -5,8 +5,10 @@
 import os
 from typing import Any, Dict, List, Optional, Set, Union, cast
 
-from rich import print
-from tabulate import tabulate  # TODO: switch to rich tables
+from rich import print as rich_print
```

Comment:
why?

Reply:
so that it does not get confused with ´print´. (the current ´main´ branch accidentally uses standard ´print´ in a few places where it ought to use rich's `print`)